IFC4x3/Properties/s/SwellingIndex_2gc$MqEfT7o9RCMSQRED$B/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="SwellingIndex_2gcMqEfT7o9RCMSQREDB" Name="SwellingIndex" UniqueId="aa9bf5b4-3a97-47c8-96cc-59c69b38dfcb" PrimaryDataType="IfcReal" />
```

Comment:
Ratio measure or IfcReal?

Reply:
IfcReal in excel spec. But let's ask @j-weil.

Reply:
IfcRatioMeasure.